test/unit/spec/PasswordComplexityUtil_spec.js

```diff
@@ -0,0 +1,293 @@
+define([
+  'okta',
+  'okta/underscore',
+  'util/PasswordComplexityUtil'
+],
+function (Okta, _, PasswordComplexityUtil) {
+
+  describe('PasswordComplexityUtil', function () {
+
+    describe('default model properties', function () {
+      beforeEach(function () {
+        this.defaultProps = PasswordComplexityUtil.modelProps;
+      });
+
+      it('has 6 properties', function () {
+        expect(_.size(this.defaultProps)).toEqual(6);
+      });
+
+      it('minLength is 8', function () {
```

Comment:
not only checking the size, but more checking all the default props.
Maybe something like ```minLenght is a number, not required and with default size 8```
Same comment for the other tests